dpipe/im/grid.py

```diff
@@ -50,6 +50,27 @@ def get_boxes(shape: AxesLike, box_size: AxesLike, stride: AxesLike, axis: AxesL
         yield make_box_([start, np.minimum(start + box_size, shape)])
 
 
+def make_batch(divide_iterator: Iterable, batch_size: int = 1):
+    patches_to_batch = []
+    n = 0
+    for patch in divide_iterator:
+        patches_to_batch.append(torch.from_numpy(patch))
+        n += 1
+        
+        if n == batch_size:
+            n = 0
+            yield torch.cat(patches_to_batch).numpy()
```

Comment:
Why u need numpy -> torch -> numpy conversions?

Reply:
Torch works faster than numpy, but inference_step expects numpy.ndarray, not torch.Tensor, so it is